packages/lodestar/src/cli/commands/interop.ts

```diff
@@ -116,23 +122,32 @@ export class InteropCommand implements ICliCommand {
     } else {
       peerId = createPeerId();
     }
+    const network: {multiaddrs?: string[]; bootnodes?: string[]} = conf.network || {};
+    const multiaddrs = network.multiaddrs || [];
```

Comment:
default multiaddrs should be ["/ip4/127.0.0.1/tcp/30606"] but wait for caymans rsponse down below

Reply:
those options are only available i side network module, thats why im asking to move libp2p stuff inside network module